<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);
background-color: rgb(212, 212, 212);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="currentWindHumidity">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>humudity</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTemperature">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentLocation">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Istanbul</string>
    </property>
   </widget>
   <widget class="QLabel" name="MinTemp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>mintemp</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxTemp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>maxtemp</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ruzgarhizi">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>windspeed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Guncelle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string>GÃ¼ncelle</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>861</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <widget class="QLabel" name="temp_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_first">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>a</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_first">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Now</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_7">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Today</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_8">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_8">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_7">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_first_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_first_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>c</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_first_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>c</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_first_5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>80</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>c</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_first_6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>c</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_first_7">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>80</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>c</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_first_8">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>70</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>c</string>
     </property>
    </widget>
    <widget class="QLabel" name="CurrentTime">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CurrentTime</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>531</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 180, 180);</string>
    </property>
    <widget class="QLabel" name="minmax_cuma">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min/max</string>
     </property>
    </widget>
    <widget class="QLabel" name="DateCuma">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="DateSali">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="Daycars">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ã§arÅamba</string>
     </property>
    </widget>
    <widget class="QLabel" name="DatePrs">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="Daypers">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>perÅembe</string>
     </property>
    </widget>
    <widget class="QLabel" name="Daysali">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>salÄ±</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_prs">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
    <widget class="QLabel" name="DateCrs">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="minmax_sali">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min/max</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_cuma">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_crs">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>90</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
    <widget class="QLabel" name="minmax_pers">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>156</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>min/max</string>
     </property>
    </widget>
    <widget class="QLabel" name="Daycuma">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cuma</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_sali">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>90</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
    <widget class="QLabel" name="minmax_cars">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min/max</string>
     </property>
    </widget>
    <widget class="QLabel" name="Datepzt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="minmax_pzt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min/max</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_pzt">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>90</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
    <widget class="QLabel" name="Daypzt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>pazartesi</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="currentWeatherIcon">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
